### PR TITLE
Request response payload capture verify by  smoke test

### PR DESCRIPTION
**Pull request by samarth-gupta-traceable**
Request response payload capture verify by  smoke test

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/AppServerTest.groovy [hunk 1]
```diff
@@ -93,12 +90,68 @@ abstract class AppServerTest extends SmokeTest {
...
+    traces.countFilteredAttributes("http.request.body") == 1
+
+    and: "Request body should be same as sent content"
+    traces.getFilteredAttributeValue("http.request.body") == stringBody
+
+    and: "Response body should be same as sent content"
+    traces.getFilteredAttributeValue("http.response.body") == stringBody
+
+    and: "Response body should be same as received response body"
+    traces.getFilteredAttributeValue("http.response.body") == responseBody
```
Comment (pavolloffay) [inline smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/AppServerTest.groovy:142]: this seems to be duplicated

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/GlassFishSmokeTest.groovy [hunk 2]
```diff
@@ -1,4 +1,45 @@
...
 package org.hypertrace.agent.smoketest
 
-class GlassfishSmokeTest {
+import spock.lang.Ignore
+
+import java.time.Duration
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.containers.wait.strategy.WaitStrategy
+
+@Ignore
```
Comment (pavolloffay) [inline smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/GlassFishSmokeTest.groovy:14]: please create an issue and add a link here

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/TomeeSmokeTest.groovy [hunk 3]
```diff
@@ -5,17 +5,20 @@
 
 package org.hypertrace.agent.smoketest
 
+import spock.lang.Ignore
+
 import java.time.Duration
 import org.testcontainers.containers.wait.strategy.Wait
 import org.testcontainers.containers.wait.strategy.WaitStrategy
 
+@Ignore
```
Comment (pavolloffay) [inline smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/TomeeSmokeTest.groovy:14]: is tomee also failing? If so create an issue

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/GlassFishSmokeTest.groovy (hunk 2 above)
Comment (samarth-gupta-traceable) [reply smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/GlassFishSmokeTest.groovy:14]: https://github.com/hypertrace/javaagent/issues/276

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/TomeeSmokeTest.groovy (hunk 3 above)
Comment (samarth-gupta-traceable) [reply smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/TomeeSmokeTest.groovy:14]: the v.15.0 release did not have test for `tomee` . However it is present in latest code. 
The the image matrix that is currently published is based on v.15.0 release so tomee test container images are missing in docker hub. 
if you look for changes in current PR in file: smoke-tests/matrix/build.gradle
I have added config to publish the same. 
Once this PR is merged, images will be published and I will create a follow up PR to enable this with new image tags.

File: smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/AppServerTest.groovy (hunk 1 above)
Comment (samarth-gupta-traceable) [reply smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/AppServerTest.groovy]: removed

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**